README.md

```diff
@@ -8,196 +8,29 @@ about the [Universal AST](https://doc.bblf.sh/) of the code itself. gitbase is b
 gitbase implements the *MySQL* wire protocol, it can be accessed using any MySQL
 client or library from any language.
 
+[src-d/go-mysql-server](https://github.com/src-d/go-mysql-server) is the project where the SQL engine used by ***gitbase*** is implemented.
+
 ## Status
 
 The project is currently in **alpha** stage, meaning it's still lacking performance in a number of cases but we are working hard on getting a performant system able to process thousands of repositories in a single node. Stay tuned!
 
 ## Examples
 
-To see the SQL subset currently supported take a look at [this list](https://github.com/src-d/go-mysql-server/blob/0084abf48137b4d72c6f948abfde91a00f3f77f0/SUPPORTED.md) from [src-d/go-mysql-server](https://github.com/src-d/go-mysql-server).
-
-[src-d/go-mysql-server](https://github.com/src-d/go-mysql-server) is the project where the SQL engine used by ***gitbase*** is implemented.
-
-#### Get all the HEAD references from all the repositories
-
-```sql
-SELECT * FROM refs WHERE ref_name = 'HEAD'
-```
-
-#### Commits that appears in more than one reference
+You can see some examples [here](/docs/using-gitbase/examples.md).
```

Comment:
I'd rather had a more descriptive link name than `here`. Also it looks strange to have two small one phrase paragraphs. Either have a list:

```markdown
* [Query examples](/docs/using-gitbase/examples.md)
* [Main documentation](/docs)
```

Or a phrase with both links:

```markdown
You can see some [query examples](/docs/using-gitbase/examples.md) in [gitbase documentation](/docs).
```